lib/ts/claims/primitiveArrayClaim.ts

```diff
@@ -33,6 +33,11 @@ export class PrimitiveArrayClaim<ValueType> {
             id?: string
         ): SessionClaimValidator => {
             const DateProvider = DateProviderReference.getReferenceOrThrow().dateProvider;
+            if (maxAgeInSeconds && maxAgeInSeconds < DateProvider.getThresholdInSeconds()) {
```

Comment:
no implicit checks! change it to maxAgeInSeconds !== undefined. What if the user puts maxAgeInSeconds to 0? Then if (maxAgeInSeconds) will be false.

Reply:
Thanks for catching it. I have fixed it. There must an eslint rule to prevent this.. but I we don't seem to be using eslint in this project currently.